src/utransport.rs

```diff
@@ -656,4 +674,26 @@ mod tests {
         let check_on_receive_res = up_client_foo.check_on_receive(&uuri_1, &umessage);
         assert!(check_on_receive_res.is_err());
     }
+
+    #[test]
+    fn test_comparable_listener_pointer_address() {
+        let bar = Arc::new(ListenerBar);
+        let comp_listener = ComparableListener::new(bar);
+        println!("comp_listener: {:?}", comp_listener);
```

Comment:
Do these println! statements contribute anything to the test?

Reply:
They let me easily see the result if I run with --nocapture to "prove" the Debug implementation works :slightly_smiling_face: 

Would it be better to use logging crate (tracing?) and then env_logger to dev-dependencies?

Reply:
I suppose I could do some string manipulation ensure we get the intended Debug implementation too, I suppose.

Which of those sounds reasonable?

Reply:
Well, I have stated before that IMHO expecting the output of Display::display to contain any particular text is an antipattern.

That said, if you want to make sure that _something_ is produced, you could do something like
```rust
let bar = Arc::new(ListenerBar);
let comp_listener = ComparableListener::new(bar);
assert!(!format!("{}", comp_listener).is_empty());
```

Reply:
Okay. I'll go with this.